PhoneBook/src/main/java/humphrey/jeriah/PhoneBook.java

```diff
@@ -0,0 +1,52 @@
+package humphrey.jeriah;
+import java.util.*;
+
+/**
+ * Created by jeriahhumphrey on 2/7/17.
+ */
+public class PhoneBook {
+
+     TreeMap <String, String> phoneBook = new TreeMap();
+
+
+     public String lookUp(String key){
+         return phoneBook.get(key);
+     }
+
+     public void addEntry (String name, String number){
+         phoneBook.put(name, number);
+
+     }
+
+     public TreeMap getMap(){
```

Comment:
Production code that exists for testing purposes only is [usually something to avoid](http://xunitpatterns.com/Test%20Logic%20in%20Production.html) -- it's referred to as a code smell.